dist/Safe/t/safetie.t

```diff
@@ -0,0 +1,23 @@
+#!perl
+
+BEGIN {
+    require Config;
+    import Config;
+    if ($Config{'extensions'} !~ /\bOpcode\b/) {
+	print "1..0\n";
+	exit 0;
+    }
+}
+
+use strict;
+use warnings;
+use Test::More;
+use Safe;
+plan(tests => 1);
+
+my $c = new Safe;
```

Comment:
We've been trying to get away from indirect object notation.  Could this be rewritten as `my $c = Safe->new();`?

Reply:
I removed this file now. Let me know if you want me to update the existing tests.

Reply:
> I removed this file now. Let me know if you want me to update the existing tests.

No, there are others who are more concerned with this than I am.  We'll let them follow up in a subsequent pull request. :-)